src/coreclr/gc/gcpriv.h

```diff
@@ -914,7 +914,7 @@ class allocator
         {
             alloc_list* al = &(allocator_to_copy->alloc_list_of (i));
             alloc_list_tail_of(i) = al->alloc_list_tail();
-#if !defined(TARGET_AMD64) && !defined(TARGET_X86)
+#if !defined(TARGET_AMD64) && !defined(TARGET_X86) && !defined(TARGET_BROWSER)
```

Comment:
If we do the change in gcenv.base.h based on the runtimelabs stuff, this change is not needed.